admin-api/how-to-use.md

```diff
@@ -0,0 +1,280 @@
+        "2": "Visitor - Edited"
+    }
+}
+```
+
+Full `curl` request:
+
+```bash
+curl --location --request PUT 'http://yourdomain.test/admin-api/customers/group/1' \
+--header 'Content-Type: application/json' \
+--header 'Authorization: Bearer YOUR_ACCESS_TOKEN' \
+--data '{
+    "localizedNames": {
+        "1": "Odwiedzający - Edytowany",
+        "2": "Visitor - Edited"
+    }
+}'
+```
+
+If everything goes well, you should receive a response with information about the user group. The response should look like this:
+
+```json
+{
+    "customerGroupId": 1,
+    "localizedNames": {
+        "1": "Odwiedzający - Edytowany",
+        "2": "Visitor - Edited"
+    },
+    "reductionPercent": 0.0,
+    "displayPriceTaxExcluded": false,
+    "showPrice": true,
+    "shopIds": [
+        1
+    ]
+}
+```
+
+Notice that the user group name has been changed. You can now use the same method to edit other resources in the API. Let's now proceed to the next step.
+
+## Adding a new resource
+
+Now that you know how to retrieve and edit information about a single resource, you can try to add a new resource. To do that, you must send a `POST` request to the `/admin-api/customers/group` endpoint. You need to provide the access token in the `Authorization` header and the data you want to add to the request body.
+
+You can send the following JSON data to add a new user group:
+
+```json
+{
+    "localizedNames": {
+        "1": "Nowa grupa",
+        "2": "New group"
+    },
+    "reductionPercent": 0.0,
+    "displayPriceTaxExcluded": false,
+    "showPrice": true,
+    "shopIds": [
+        1
+    ]
+}
+```
+
+Full `curl` request:
+
+```bash
+curl --location 'http://yourdomain.test/admin-api/customers/group' \
+--header 'Content-Type: application/json' \
+--header 'Authorization: Bearer YOUR_ACCESS_TOKEN' \
+--data '{
+    "localizedNames": {
+        "1": "Nowa grupa",
+        "2": "New group"
+    },
+    "reductionPercent": 0.0,
+    "displayPriceTaxExcluded": false,
+    "showPrice": true,
+    "shopIds": [
+        1
+    ]
+}'
+```
+
+This time, you should receive a response with information about the newly created user group. The response should look like this:
+
+```json
+{
+    "customerGroupId": 4,
+    "localizedNames": {
+        "1": "Nowa grupa",
+        "2": "New group"
+    },
+    "reductionPercent": 0.0,
+    "displayPriceTaxExcluded": false,
+    "showPrice": true,
+    "shopIds": [
+        1
+    ]
+}
+```
+
+and you will see the new user group in the PrestaShop back office.
+
+![New group added via the API available in the back office](../img/bo_groups.jpeg)
+
+If you don't provide all the necessary information, you will receive an error message. For example, if you try to add a new user group without providing the `localizedNames` parameter, you will receive the following response:
+
+![Error when adding a new resource without providing required information](../img/postman_add_group_error.png)
```

Comment:
Why do you put a screenshot instead of copying/pasting the error in raw text? (useful for search engines)

Reply:
This whole page (and some part about creating Api Client) have been copied from the blog So for now I didn't plan on imp^roving them, I also need to confirm with @kpodemski that this part is ok in the dev doc

I think the modification you suggest can be done in a later PR